src/Components/Subscriptions/Table/index.jsx

```diff
@@ -100,16 +107,16 @@ const Table = ({ data }) => {
                 )}
                 <td className={`${styles.item} ${styles.td}`}>{formatDate(subscription.date)}</td>
                 <td
-                  className={`${styles.btnEdit} ${styles.td}`}
                   onClick={() => handleEdit(subscription._id)}
```

Comment:
I know it won't work until we merge the branches, but this is not necessary anymore. 
This week, this button should change the route to /subscriptions/form, you can wrap the button with a link instead, for example. 
Maybe you want to make this change when I make the PR. I think tomorrow I will have it, sorry for the delay...